test/verification_test.rb

```diff
@@ -127,19 +127,6 @@ def test_using_symbol_back_redirects_to_referrer
     assert_redirected_to '/foo'
   end
 
-  def test_no_deprecation_warning_for_named_route
```

Comment:
Why did these tests need to be removed?

Reply:
One is testing a deprecation warning that is specific to a certain version of rails. Another is testing a specific feature of rails that uses the _method param to map post requests to update and destroy. Think rails still does this, but the way it works was changed so the test fails.